docs/architecture.md

```
@@ -0,0 +1,127 @@
+## 🚀 快速开始
+
+```bash
+# 安装依赖
+yarn install
+
+# 构建项目
+yarn build
+
+# 启动示例
+yarn workspace @onekeyfe/connect-examples start
+```
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)